app/controllers/case_contacts_controller.rb

```diff
@@ -181,6 +185,12 @@ def create_case_contact_for_every_selected_casa_case(selected_cases)
     end
   end
 
+  def send_reimbursement_email(case_contact)
+    if case_contact.want_driving_reimbursement_changed? && case_contact.want_driving_reimbursement? && !current_user.supervisor.blank?
+      SupervisorMailer.reimbursement_request_email(current_user, current_user.supervisor).deliver
```

Comment:
```suggestion
      SupervisorMailer.reimbursement_request_email(current_user, current_user.supervisor).deliver_later
```

Reply:
Could you also add/modify `spec/requests/case_contacts_spec.rb` and add an expectation that `SupervisorMailer.reimbursement_request_email` is called when those conditions are met, and another expection that it is not called when the conditions are not met?

Reply:
Thank you for the feedback. Have done the required changes and pushed the updated code.